src/app/myworkflows/my-workflow/my-workflow.component.html

```diff
@@ -0,0 +1,85 @@
+  ~    You may obtain a copy of the License at
+  ~
+  ~        http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~    Unless required by applicable law or agreed to in writing, software
+  ~    distributed under the License is distributed on an "AS IS" BASIS,
+  ~    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~    See the License for the specific language governing permissions and
+  ~    limitations under the License.
+  -->
+
+  <app-header>
+      <img src="../assets/images/dockstore/dockstore-workflows-green.png"> My Workflows
+      <div class="pull-right refresh-all-holder">
+        <button type="button" class="refresh-all-button btn btn-primary" (click)="refreshAllWorkflows()" [disabled]="refreshMessage !== null">
+          Refresh All Workflows
+        </button>
+      </div>
+    </app-header>
+    <div class="container padding-top">
+      <app-alert></app-alert>
+      <div class="row" *ngIf="orgWorkflows?.length == 0">
+        <div class="col-md-12">
+          <div class="alert alert-info" role="alert">
+            <p>
+              <span class="glyphicon glyphicon-info-sign"></span>
+              There are currently no workflows registered under this account, to add your first workflow, do one of the following:
+            </p>
+            <ul>
+              <li>
+                Press the 'Refresh All Workflows' button to create workflow stubs for all of your source control repositories.  Browse to the workflow stubs and refresh/publish those that correspond to real workflows.
+              </li>
+              <li>
+                Press the 'Register Workflow' button to manually add a workflow.
+              </li>
+            </ul>
+            <div *ngIf="!hasGitHubToken">
+              <a class="btn btn-block btn-social btn-github" (click)="link()">
+                <span class="fa fa-github"></span> You do not have a linked GitHub token, click here to link a GitHub token
+              </a>
+            </div>
+          </div>
+        </div>
+      </div>
+      <!--Tool Content Page-->
+      <div class="row">
+        <div class="col-md-3 containers-rsb">
+          <accordion [closeOthers]="oneAtATime">
+            <accordion-group *ngFor="let orgObj of orgWorkflows"[heading]="orgObj?.sourceControl + '/' + orgObj?.organization" panelClass="containers-accordion" [isOpen]="orgObj?.isFirstOpen">
+              <div *ngFor="let workflowObj of orgObj?.workflows">
+                <div class="panel-container-label" [ngClass]="{selected: workflow.id === workflowObj.id}">
+                  <div class="container-name-oflw pull-left" title="{{workflowObj?.repository + (workflowObj?.workflowName ? ' (' + workflowObj?.workflowName + ')' : '')}}">
+                    <a style="cursor: pointer;" (click)="goToWorkflow(workflowObj)">
+                      {{workflowObj?.repository + (workflowObj?.workflowName ? ' (' + workflowObj?.workflowName + ')' : '')}}
```

Comment:
Make sure this merges properly to the new format with a slash instead of brackets.